.github/workflows/performance.yml

```diff
@@ -72,8 +72,27 @@ jobs:
           cat >> $GITHUB_OUTPUT << EOF
           marker_expressions=${MARKER_EXPRESSIONS}
           EOF
-
-          AP_MODELS="${{ github.event.inputs.ap_models || 'cig_wf188n,cig_wf196,hfcl_ion4xe,yuncore_fap655,yuncore_ax820,edgecore_oap101-6e,edgecore_eap102,edgecore_eap101,edgecore_eap104,cig_wf186w' }}"
+          
+          DOW=$(date +%u)
+          HOD=$(date +%H)
+          SLOT=$(( 10#$DOW % 7))
+          echo "Current date is $(date --iso-8601=s)"
+          echo "Current date of week is $DOW which is $(( 10#$DOW % 7 )) in cron notation"
+          echo "Current time is $HOD hours, which is $(( 10#$HOD % 24)) in cron notation"
+          echo "Current slot is $SLOT"
+
```

Comment:
Maybe add a comment here describing what you're doing here. Could be useful for somebody looking through here.

Reply:
Sure @carsten989 , will add the comments and check in .